package.json

```diff
@@ -16,6 +16,7 @@
     "motion-ui": "~1.2.2"
   },
   "devDependencies": {
+    "autoprefixer": "^8.1.0",
```

Comment:
Why ? We already have https://github.com/zurb/foundation-zurb-template/blob/1755a0060d932b02e96c38784b9fc0987a2e17bc/package.json#L25

Reply:
gulp-autoprefixer does not load it (anymore) even if it is installed.

Reply:
See https://github.com/sindresorhus/gulp-autoprefixer/issues/98

Reply:
But we should probably upgrade to gulp-autoprefixer 5.0.0. In general the whole Gulp ecosystem is a nightmare.

Reply:
From when ?
It's not like we updated `gulp-autoprefixer` recently.

Reply:
> In general the whole Gulp ecosystem is a nightmare.

I couldn’t agree more

Reply:
> From when ?
> It's not like we updated gulp-autoprefixer recently.

Did a fresh instal and got the error that autoprefixer can not be loaded.